src/renderer/CanvasRenderer.js

```diff
@@ -239,6 +239,10 @@ export default class CanvasRenderer implements RenderTarget<HTMLCanvasElement> {
                     text.bounds.top + text.bounds.height
                 );
             }
+            this.ctx.shadowColor = '';
+            this.ctx.shadowOffsetX = 0;
+            this.ctx.shadowOffsetY = 0;
+            this.ctx.shadowBlur = 0;
```

Comment:
These 4 shadow properties needed resetting after they were used in the forEach, because otherwise all the following elements in the Tree were rendering undesired shadow.